### PR TITLE
Release for S161

### PR DESCRIPTION
**Pull request by isra-fel**
## Description

I want to use a branch `release-temp` for ado pipeline, so please approve this PR directly.
I'll submit another PR from `release-temp` to `release-2019-11-26`. That one should be reviewed. Thanks.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by isra-fel** [comment]
@msJinLei @wyunchi-ms could you review this please?
One thing I notice is that resource manager module generated 2 serialized json. Do you know why?

File: tools/Az/Az.psd1 [hunk 1]
```diff
@@ -64,18 +64,18 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.6.4'; },
                @{ModuleName = 'Az.Billing'; RequiredVersion = '1.0.1'; },
```
Comment (erich-wang) [inline tools/Az/Az.psd1]: missing modules for Az.DataBoxEdge and Az.SqlVirtualMachine

File: ChangeLog.md [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+    - Add EnableAutomaticRepair, AutomaticRepairGracePeriod, and AutomaticRepairMaxInstanceRepairsPercent parameters to the following cmdlets:
+        New-AzVmssConfig
+        Update-AzVmss
+* Cross tenant gallery image support for New-AzVM
+* Add 'Spot' to the argument completer of Priority parameter in New-AzVM, New-AzVMConfig and New-AzVmss cmdlets
+* Add DiskIOPSReadWrite and DiskMBpsReadWrite parameters to Add-AzVmssDataDisk cmdlet
+* Change SourceImageId parameter of New-AzGalleryImageVersion cmdlet to optional
+* Add OSDiskImage and DataDiskImage parameters to New-AzGalleryImageVersion cmdlet
+* Add HyperVGeneration parameter to New-AzGalleryImageDefinition cmdlet
+
```
[the remark this answers is not in the document]
Comment (isra-fel) [reply ChangeLog.md]: Added change log for `DataBoxEdge`. `SqlVirtualMachine` had no code change, so I put it on the top.

File: tools/Az/Az.psd1 (hunk 1 above)
Comment (isra-fel) [reply tools/Az/Az.psd1]: Added. Thanks

**Pull request merged**